operations/research-and-development/organization/growth.md

```diff
@@ -5,21 +5,18 @@ The Growth team is responsible for accelerating usage and revenue by optimizing
 ## Areas of Ownership
 
 - Activation
+  - First-time user onboarding experience
+  - Onboarding Templates
+  - User Invites
   - Signup-to-login, install-to-login
   - User invitation flows
   - Install-to-dau
-  - Free-to-trial
-  - Trial-to-paid
 - Adoption
-  - First-time user onboarding experience
   - Day 1, Day 7, Day 28 retention
   - DAU Nuturing
+- Growth Telemetry
 - User Feedback & Announcements
   - NPS system
   - Feedback widgets
   - In-product announcements
-  - Admin Advisor
-- Expansion
-  - In-product upsells
-- Packaging & Pricing
-- Growth Telemetry
+  - Admin Advisor
```

Comment:
Hasn't this been removed?

Reply:
@marianunez ?

Reply:
Yes, the original Admin Advisor was disabled. The concept of suggesting specific features depending on usage is still discussed but I'll remove this and the new features are considered under upsells that Self-Serve team owns.